### ISSUE TITLE
Hi I am able to compile the file. 

### ISSUE THREAD

**Issue by cliffordx3** [Mst1k/CS1-6NoSteamMultiHack#1]
Hi i am successfully compile the file, but the problem is i can't update the offsets for it, 

i am playing cs 1.3 do you have offsets for cs 1.3?
i cannot figure out how to get this value from cheat engine.

localplayer:
entites:
view matrix:

any reply would be so much appreciated, thanks in advanced!

**Comment by Mst1k** [comment]
well, i dont exactly know if these game interfaces are same in cs 1.3, but i will research when have free time, ill let you know

**Comment by cliffordx3** [comment]
i think its the same, the problem is when i search for value i like localplayer it seems that does not have offsets but the value alone. when try to dissect data structures using cheat engine, the offsets was 0000

**Comment by cliffordx3** [comment]
[https://i.imgur.com/WQZdbZN.jpeg](url)

Take a look at i uploaded screenshot so you can see.

**Comment by Mst1k** [comment]
try finding cl_entities, it doesnt contain health inside, just position, also you will need to find the structures that contains dormant and team values, about reseearching it for you, it will take me along time to find that offset, with i dont have right now

**Comment by cliffordx3** [comment]
Yeah. i think i should but how can i find cl_entities i mean where should i start and how do i find it? do i have to use cheat engine or other debugging tools? thank you.

**Comment by cliffordx3** [comment]
i think if you have c.s 1.3 files. you can send me the same offsets because its the same client.

**Comment by Mst1k** [comment]
bro the entity list is an array of objects, not an array of pointers, the objects are a huge size, i dont remember ok, it was arround 0x800

**Comment by cliffordx3** [comment]
yeah, but what im tellingnyou is. i have thw entitybase address, but cant figure out why i cant find the offset of the entitylist of an localteam. thank you.

**Comment by Mst1k** [comment]
understand, there is a CBaseEntity i think you have this, and know that this is just for the localplayer, not all players have one of it, istead you want to find cl_entitys

**Comment by cliffordx3** [comment]
yeah i want to find the cl_entitys.

**Comment by cliffordx3** [comment]
> understand, there is a CBaseEntity i think you have this, and know that this is just for the localplayer, not all players have one of it, istead you want to find cl_entitys

Hi i uploaded a files contains the c.s Its just 100+ mb  [.](https://drive.google.com/file/d/1Z9Z6zageDjsICcFnmn6VdonwzHipmyGs/view) Maybe you could help me get into this offset so i can test my code thank you. any help would be so much appreciated. 

localplayer:
entites:
view matrix:
iteam:
ihealth:

**Comment by Mst1k** [comment]
How do you start? do reverse engineering, i cant teach you that, make your hipotesis, and use your intuision.

**Comment by cliffordx3** [comment]
> i just dont have the time to help you

but you show me some tutorial?

**Comment by Mst1k** [comment]
i dont have tutorials

**Comment by cliffordx3** [comment]
how about get me those adress and offset i am willing to pay you.
